docs/pages/identity-security/usage/alerts.mdx

```diff
@@ -0,0 +1,140 @@
+
+## Overview
+
+Identity Activity Center alerts help you detect and respond to security threats by monitoring for:
+- Unusual authentication patterns
+- Privilege escalations
+- Configuration changes that affect security
+- Account compromises
+- Policy violations
+
+All detections are pre-configured and currently cannot be modified. The severity levels (Critical, High, Medium, Low) are defined by the Identity Security Team based on the potential impact of each detection.
+
+![Alerts](../../../img/access-graph/alerts.png)
+
+
+## Detection Categories
+
+### AWS Detections
+
+Identity Activity Center monitors your AWS infrastructure for critical security events:
+
+**Critical & High Severity:**
+- Root account activities (console login, access key creation, general activity)
+- Security settings deletions (CloudTrail, GuardDuty, flow logs)
+- EBS encryption disabled
+- DB snapshots made public
+- Exposed credential policy modifications
+- Login profile updates
+
+**Medium & Low Severity:**
+- IAM user creation
+- DB snapshot attribute modifications
+- AWS key deletion or disabling
+
+### GitHub Detections
+
+Monitor your GitHub organizations and repositories for security-relevant changes:
+
+**Critical & High Severity:**
+- Organization security updates (SAML, 2FA, OAuth restrictions)
+- Protected branch policy overrides or deletions
+- Repository access and visibility changes
+- Secret scanning alerts
+- Organization member updates
+
+**Medium & Low Severity:**
+- Repository member updates
+- Organization moderator additions
+- Integration installations
+
+**Advanced Security Detections:**
+The system includes 26 sub-types of GitHub advanced security change detections that monitor for modifications to security features like Dependabot alerts.
+
+### Okta Detections
+
+Track identity and access management events in your Okta environment:
+
+**High Severity:**
+- Admin MFA disabled
+- All MFA factors reset
+- OAuth token reuse patterns
+- Rate limit violations
+- Sign-on policy evaluations
+- IDP lifecycle changes
+- API token creation/revocation
+- Security threat events
+
+**Medium Severity:**
+- Excessive MFA failures
+- Multiple password reset attempts
+- Dormant account access
+- Support-initiated password or MFA resets
+
+### Teleport Detections
+
+Monitor Teleport-specific security events:
+
+**High Severity:**
+- Root SSH session initiation
+- Authentication without MFA for local accounts
+- Unusual authentication failure patterns
+- Role creation/updates/deletions
+- Connector updates
+- Unusual session commands
+
+### Cross-Platform Detections
+
+**Impossible Travel Detection** (GitHub, Okta, Teleport):
+- Severity: High
+- Triggers when login attempts occur from multiple geographic locations within a timeframe that makes legitimate travel physically impossible
+
+## Alert Management
+
+### Current Capabilities
+- View alerts across all integrated systems
+- Filter by severity level
+- Investigate related events
```

Comment:
I would expand each of these into a sentence. Otherwise, I'm not sure what each item means in terms of what I can do with the Web UI.